internal/provider/gce/internal/google/network.go

```diff
@@ -13,8 +13,9 @@ import (
 )
 
 const (
-	NetworkDefaultName = "default"
-	NetworkPathRoot    = "global/networks/"
+	NetworkDefaultName  = "default"
+	NetworkPathRoot     = "global/networks/"
+	ExternalNetworkName = "External NAT"
```

Comment:
You changed a bunch of places to use the string literal. You might as well use this const in those places.

Reply:
The string literal is only used in tests, which is always my preference for best practice. I prefer to see broken tests if the const ever changes so things can be double checked.